code/modules/projectiles/guns/ds13/stasis.dm

```diff
@@ -0,0 +1,142 @@
+/obj/item/weapon/gun/energy/stasis
+	name = "Stasis Gun"
+	desc = "Wow! You've managed to obtain it!"
+	icon = 'icons/obj/weapons/ds13guns48x32.dmi'
+	icon_state = "forcegun"
+	item_state = "forcegun"
+	fire_sound = 'sound/weapons/Taser.ogg'
+	fire_sound_text = "stasis blast"
+	w_class = ITEM_SIZE_HUGE
+	charge_cost = 5
+	cell_type = /obj/item/weapon/cell/potato //Potato cell is underrated
+	projectile_type = /obj/item/projectile/bullet/stasis
+	removeable_cell = FALSE
+	safety_state = FALSE
+	shot_volume = VOLUME_QUIET
+
+/obj/item/weapon/gun/energy/stasis/military
+	recharge_time = 12
```

Comment:
Assuming this is in seconds, its wayyyyyy too low.

The fully upgraded stasis module in DS2 with everything perfect takes 40 seconds per point, but even that is extreme. Normal upgrades get you to 100 seconds, unupgraded is much longer, several minutes i believe


This charging is a longterm thing, something that happens in downtime between fights. i'sd say go with a time of two minutes